core/src/main/scala/cats/conversions/VarianceConversions.scala

```diff
@@ -1,10 +1,14 @@
 package cats
 package conversions
+import cats.arrow.Profunctor
 
 trait VarianceConversions extends VarianceConversionsLowPriority {
   implicit def autoWidenBifunctor[F[_, _]: Bifunctor, A, B >: A, C, D >: C](fac: F[A, C]): F[B, D] =
     Bifunctor[F].leftWiden(Bifunctor[F].rightFunctor.widen(fac))
 
+  implicit def autoWidenProfunctor[F[_, _]: Profunctor, A, B <: A, C, D >: C](fac: F[A, C]): F[B, D] =
```

Comment:
i think the name of this method should reflect the fact that it does not only widen, but narrow, too.

Reply:
The idea as I understood is that we get the behaviour `F[A, C] <: F[B, D]` given constraints `A >: B` and `C <: D`, the opposite however won't hold with these them.

The same is true for other implicits here, if the return type is `A` then we can return any instance of `A` or it's subtypes